sdk/cosmosdb/cosmos/src/queryExecutionContext/Aggregators/MaxAggregator.ts

```diff
@@ -20,11 +25,13 @@ export class MaxAggregator implements IAggregator<number> {
    * @instance
    * @param other
    */
-  public aggregate(other: number) {
+  public aggregate(other: MaxAggregateResult) {
     if (this.value === undefined) {
-      this.value = other;
-    } else if (this.comparer.compareValue(other, typeof other, this.value, typeof this.value) > 0) {
-      this.value = other;
+      this.value = other.max;
```

Comment:
We need one test where min / max is undefined. 

You can test that with the query:

```
SELECT VALUE MIN(c.blah)
FROM c
WHERE false
```

You also need another where it's undefined, but the filter matched some results. To get that case you just need to insert an array or object.

Reply:
Added